zstd_vendor/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Could this be a separate PR?

Reply:
Nope, required so `zstd_vendor` can use a specific feature of `ExternalProject_Add` that allows you specify the directory of the `CMakeLists.txt` (since it's not in the root dir for zstd).